src/gui/mzroll/alignmentdialog.cpp

```diff
@@ -143,11 +143,18 @@ void AlignmentDialog::intialSetup()
     samplesBox->clear();
     samplesBox->addItem("Select Randomly",
 						QVariant(QVariant::fromValue(static_cast<void*>(nullptr))));
+    vector<mzSample*> selectedSample;
     for(auto sample: _mw->samples) {
         if(sample->isSelected)
-            samplesBox->addItem(sample->sampleName.c_str(),
-								QVariant(QVariant::fromValue(static_cast<void*>(sample))));
+            selectedSample.push_back(sample);
```

Comment:
There is a method in `MainWindow` called `getVisibleSamples`. We can use that here.

Reply:
Also name the variable `selectedSamples` <- plural.